packages/story-editor/src/components/canvas/editElement.js

```diff
@@ -17,80 +17,62 @@
 /**
  * External dependencies
  */
+import { useState, forwardRef } from '@googleforcreators/react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { useUnits } from '@googleforcreators/units';
 
 /**
  * Internal dependencies
  */
-import { useState, useRef } from '@googleforcreators/react';
 import { getDefinitionForType } from '../../elements';
 import {
   elementWithPosition,
   elementWithSize,
   elementWithRotation,
 } from '../../elements/shared';
-import SingleSelectionMoveable from './singleSelectionMoveable';
 
 const Wrapper = styled.div`
   ${elementWithPosition}
   ${elementWithSize}
-	${elementWithRotation}
-	pointer-events: initial;
+  ${elementWithRotation}
+  pointer-events: initial;
 `;
 
-function EditElement({ element }) {
+const EditElement = forwardRef(function EditElement(
+  { element, editWrapper, onResize },
+  ref
+) {
   const { id, type } = element;
   const { getBox } = useUnits((state) => ({
     getBox: state.actions.getBox,
   }));
 
-  const [editWrapper, setEditWrapper] = useState(null);
   // Needed for elements that can scale in edit mode.
   const [localProperties, setLocalProperties] = useState(null);
 
-  const { Edit, hasEditModeMoveable } = getDefinitionForType(type);
+  const { Edit } = getDefinitionForType(type);
   const box = getBox(
     localProperties ? { ...element, ...localProperties } : element
   );
 
-  const moveable = useRef(null);
-
-  const onResize = () => {
-    // Update moveable when resizing.
-    if (moveable.current) {
-      moveable.current.updateRect();
-    }
-  };
-
   return (
-    <>
-      <Wrapper aria-labelledby={`layer-${id}`} {...box} ref={setEditWrapper}>
-        <Edit
-          element={
-            localProperties ? { ...element, ...localProperties } : element
-          }
-          box={box}
-          editWrapper={hasEditModeMoveable && editWrapper}
-          onResize={onResize}
-          setLocalProperties={setLocalProperties}
-        />
-      </Wrapper>
-      {hasEditModeMoveable && editWrapper && (
-        <SingleSelectionMoveable
-          selectedElement={element}
-          targetEl={editWrapper}
-          isEditMode
-          ref={moveable}
-        />
-      )}
-    </>
+    <Wrapper aria-labelledby={`layer-${id}`} {...box} ref={ref}>
```

Comment:
The naming here is a bit confusing now, maybe still rename the ref to `setEditWrapper` above if it's possible?
```
const EditElement = forwardRef(function EditElement(
  { element, editWrapper, onResize },
  ref: setEditWrapper
) {
```

Reply:
But it's just using the forwarded `ref`, regardless of what that `ref` is. I could rename to `forwardedRef`, but I think it's fine as is, no?

Reply:
It's just not easy to read the file like this and understand that the editWrapper and the `ref` are the same thing. Maybe just add a comment in a next PR?